next.config.js

```diff
@@ -33,6 +39,24 @@ module.exports = (phase, { defaultConfig }) => {
 
       return config
     },
+    async headers() {
+      return [
+        {
+          source: "/(.*?)",
```

Comment:
not sure this would work with i18n (https://nextjs.org/docs/pages/api-reference/next-config-js/headers#headers-with-i18n-support)

Reply:
It should be fine.

From Nextjs documentation
>  // this gets converted to /(en|fr|de)/(.*) so will not match the top-level
> // `/` or `/fr` routes like /:path* would

There is no need to match / or /:lang because it gets automatically redirected to index (/:lang/) who will match the pattern.